src/DDoSX/Entities/DomainProperty.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace UKFast\SDK\DDoSX\Entities;
+
+use UKFast\SDK\Entity;
+
+/**
+ * @property string $id
+ * @property string $name
+ * @property int $value
```

Comment:
see comment on https://github.com/ukfast/sdk-php/pull/275 - I'd make this `mixed` as it can be a boolean, string or integer